Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/WorkflowExecutionBusiness.java

```diff
@@ -83,7 +91,8 @@ public Workflow launch(String applicationName, String applicationVersion,
 
             return new Workflow(workflowID, user.getFullName(), 
                     engine.getMode().equalsIgnoreCase("pool") ? WorkflowStatus.Queued : WorkflowStatus.Running,
-                    new Date(), null, simulationName, applicationName, applicationVersion, applicationClass);
+                    new Date(), null, simulationName, applicationName, applicationVersion, applicationClass, 
+                    engine.getMode().equalsIgnoreCase("pool") ? "ShiwaPool" : ((WebServiceEngine)engine).getAddressWS());
```

Comment:
Feel free to remove any dependence to the ShiwaPool if it carries too much untested extra code.